src/App.js

```diff
@@ -29,19 +29,22 @@ class App extends Component {
   }
 
   fetchCharacters = (number) => {
+    console.log(number)
     fetch(`https://rickandmortyapi.com/api/character?page=${number}`)
       .then(data => data.json())
-      .then(data => this.setState(prevState => ({ characters: data, favCharacters: prevState.favCharacters, currentPage: prevState.currentPage })))
+      .then(data => this.setState(prevState => ({ characters: data, favCharacters: prevState.favCharacters, currentPage: number })))
```

Comment:
Looks good! If you get a chance during refactor, see if you can move your API calls into a different file.